src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -803,9 +803,7 @@ public ResponseEntity<Void> stopVoltageInit(@Parameter(description = "Study uuid
 
     @GetMapping(value = "/studies/{studyUuid}/nodes/{nodeUuid}/voltage-init/result")
     @Operation(summary = "Get a voltage init result on study")
-    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The voltage init result"),
-            @ApiResponse(responseCode = "204", description = "No voltage init has been done yet"),
-            @ApiResponse(responseCode = "404", description = "The voltage init has not been found")})
+    @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "The voltage init result"), @ApiResponse(responseCode = "204", description = "No voltage init has been done yet"), @ApiResponse(responseCode = "404", description = "The voltage init has not been found")})
```

Comment:
no need to this change

Reply:
It's required by CheckStyle otherwise the build fail.

Reply:
I use  checkStyle provided by powsybl. It's up to date